### PR TITLE
Ordering a shard crate emits an admin warning message

### PR DESCRIPTION
**Pull request by ShiftyRail**
This should hopefully give admins enough time to act before the server blows up.
Ideally the server should not blow up at all...

Also proof : 
proof

### CODE REVIEW

**Comment by MadmanMartian** [comment]
Good in theory, bad in practice. Won't that list hold a reference to the mob as its key, thus hampering deletion of said mob?

Use the ckey

**Comment by ShiftyRail** [comment]
Ok

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -1453,6 +1455,22 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
...
+	var/span = ""
+	switch (i)
+		if (1)
+			span = "notice"
+		if (2 to 5)
+			span = "warning"
+		else
+			span = "danger"
+	message_admins("<span class='[span]'>[key_name(user)] has ordered a supermatter shard supplypack, this is his #[i] order. @[formatJumpTo(user)]</span>")
+	log_admin("[key_name(user)] has ordered a supermatter shard supplypack, this is his #[i] order. @([user.x], [user.y], [user.z])")
```
Comment (DamianX) [inline code/datums/supplypacks.dm:1473]: Doesn't `message_admins` also write to logs?

Comment (ShiftyRail) [reply code/datums/supplypacks.dm:1473]: It does, after checking, but I see this duo everywhere in the codebase
If I have to defend it in a lazy way, the `log_admin` has less `<span>` bloat and that makes it easier to review in a ban request

**Pull request merged**